src/internal/packager/helm/zarf.go

```diff
@@ -61,18 +62,21 @@ func (h *Helm) UpdateZarfAgentValues(ctx context.Context) error {
 	}
 
 	// Get the current agent image from one of its pods.
-	pods := h.cluster.WaitForPodsAndContainers(
-		ctx,
-		k8s.PodLookup{
-			Namespace: cluster.ZarfNamespaceName,
-			Selector:  "app=agent-hook",
-		},
-		nil,
-	)
```

Comment:
I do not see any reason why this should wait for pods that match the selector.

Reply:
Likely since the WaitForPodsAndContainers function was the only one that returned pods it was used for convenience

Reply:
Most likely the reason. In that case we can just ignore this.